packages/scss/src/components/button/mods.scss

```diff
@@ -68,19 +72,31 @@
 	}
 }
 
+@mixin onlyIcon {
```

Comment:
we don't add ($atRoot: 'without: rule') here ?

Reply:
There's no sub-component here, so it's not necessary. As a result, I've removed the parameter on mixin `withIcon`.